bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,22 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    def _check_imperative_enabled(self):
+        imperative_check = self.section.contents.get(
+            'shortlog_check_imperative', True)
+        return bool(imperative_check)
```

Comment:
shouldn't `return bool(self.section.get('shortlog_check_imperative', True))` do the job in one line? (Note: .contents shouldn't be needed and we should optimize for readability rather than an unnoticable performance benefit)

Second, the default `True` might change, maybe put this in a class level constant?